packages/installWeather.sh

```diff
@@ -6,25 +6,26 @@
 AUTH_KEY=$WHISK_SYSTEM_AUTH
 
 SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
-PACKAGE_HOME=$SCRIPTDIR
-source "$PACKAGE_HOME/util.sh"
+CATALOG_HOME=$SCRIPTDIR
+source "$CATALOG_HOME/util.sh"
 
 echo Installing Weather package.
 
 createPackage weather \
-    -a description "Services from IBM Weather Insights" \
-    -a parameters '[ {"name":"apiKey", "required":false, "bindTime":true} ]'
+-p bluemixServiceName "weatherinsights"
+-a description "Services from the Weather Company Data for IBM Bluemix" \
+-a parameters '[ "name":"username", "required":false,"bindTime":true}, {"name":"password", "required":false, "type":"password","bindTime":true}]'
 
 waitForAll
 
-install "$PACKAGE_HOME/weather/forecast.js" \
-    weather/forecast \
-    -a description 'IBM Weather Insights 10-day forecast' \
-    -a parameters '[ {"name":"latitude", "required":true}, {"name":"longitude", "required":true},{"name":"language", "required":false},{"name":"units", "required":false}, {"name":"timePeriod", "required":false}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true} ]' \
-    -a sampleInput '{"latitude":"34.063", "longitude":"-84.217", "apiKey":"XXX"}' \
-    -a sampleOutput '{"forecasts":[ {"dow":"Monday", "min_temp":30, "max_temp":38, "narrative":"Cloudy"} ]}'
+install "$CATALOG_HOME/weather/forecast.js" \
```

Comment:
Then replace this line with install "$PACKAGE_HOME/weather/forecast.js" \
All the other install scripts in the catalog repo have been using $PACKAGE_HOME.